polaris-migrator/src/migrations/scss-remove-unused-import/scss-remove-unused-import.ts

```diff
@@ -0,0 +1,58 @@
+import type {FileInfo, API, Options} from 'jscodeshift';
+import postcss from 'postcss';
+import type {Plugin} from 'postcss';
+
+import {createInlineComment} from '../../utilities/sass';
+import {POLARIS_MIGRATOR_COMMENT} from '../../constants';
+
+interface PluginOptions extends Options {
+  importPath?: string;
+  namespace?: string;
+}
+
+const plugin = (options: PluginOptions = {}): Plugin => {
+  const {
+    importPath = 'global-styles/legacy-polaris-v8',
+    namespace = 'legacy-polaris-v8',
+  } = options;
+  const namespacePrefix = `${namespace}.`;
+
+  return {
+    postcssPlugin: 'scss-remove-unused-imports',
+    Once(root) {
+      root.walkAtRules('use', (atRule) => {
+        const usePath = atRule.params.match(/(['"])(.*?)\1/)?.[2] ?? '';
+
+        // Didn't find the target import in file
+        if (!usePath.endsWith(importPath)) {
+          return;
+        }
+
+        const alias = atRule.params.match(/as\s*((?:[^\s;])+)/)?.[1];
```

Comment:
```suggestion
        const alias = atRule.params.match(/as\s+((?:[^\s;])+)/)?.[1];
```
Isn't at least one space required between the `as` keyword and `asNamespace`?

Reply:
Also, `;` shouldn't be present in `atRule.params`.